app/code/Magento/Ui/view/base/web/js/lib/validation/rules.js

```diff
@@ -419,13 +419,13 @@ define([
         ],
         'validate-email': [
             function (value) {
-                return utils.isEmptyNoTrim(value) || /^([a-z0-9,!\#\$%&'\*\+\/=\?\^_`\{\|\}~-]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+(\.([a-z0-9,!\#\$%&'\*\+\/=\?\^_`\{\|\}~-]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+)*@([a-z0-9-]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+(\.([a-z0-9-]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF])+)*\.(([a-z]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF]){2,})$/i.test(value);//eslint-disable-line max-len
+                return utils.isEmptyNoTrim(value) || value.indexOf('@') !== -1;
             },
             $.mage.__('Please enter a valid email address (Ex: johndoe@domain.com).')
         ],
         'validate-emailSender': [
             function (value) {
-                return utils.isEmptyNoTrim(value) || /^[\S ]+$/.test(value);
+                return utils.isEmptyNoTrim(value) || value.indexOf('@') !== -1;
```

Comment:
Please fix misprint with validate-emailSender if you don't mind (or tell me and I'll report it separately if you believe it is out of scope).